model/src/main/java/org/mskcc/cmo/metadb/model/igo/IgoQcReport.java

```diff
@@ -0,0 +1,59 @@
+package org.mskcc.cmo.metadb.model.igo;
```

Comment:
If `../QcReport` is removed then this will be renamed from `IgoQcReport` to `QcReport`